pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -320,6 +351,10 @@ func (config *V2RegistriesConf) postProcessRegistries() error {
 			if err != nil {
 				return err
 			}
+
+			if mir.Location == "" {
+				return &InvalidRegistries{s: "invalid condition: mirror location is unset"}
```

Comment:
This is fine for now (certainly better than dropping the pre-existing test case), but should be resolved along with the FIXME in `parseLocation`. Note that `unqualifiedSearchRegistries` now also accepts empty values, and shouldn’t.

Reply:
… `unqualifiedSearchRegistries` actually probably rejects them a few lines afterwards due to the regex match

Reply:
noted as FIXME.